src/erasure-code/jerasure/jerasure_init.cc

```diff
@@ -3,7 +3,7 @@
 /*
  * Ceph distributed storage system
  *
- * Copyright (C) 2014 Cloudwatt <libre.licensing@cloudwatt.com>
```

Comment:
One more thought on this issue. If the shec and jerasure plugins are loaded at the same time, don't they share the same galois field arrays? So shec would be enabled for w4 or maybe jerasure would lose w4 depending on the order of loading?